cmd/analyzer-progress/main.go

```diff
@@ -0,0 +1,58 @@
+package main
```

Comment:
I assume this might have been for testing, do you think that we can remove this and just use the change to the main analyzer command?

Reply:
Yes, exactly! The analyzer-progress wrapper was just for testing/demonstration purposes. The progress reporting flags (--progress-output and --progress-format) are already integrated into the main analyzer command (cmd/analyzer/main.go:387-388), so this wrapper is no longer needed.

I'll remove the cmd/analyzer-progress directory.